adapters/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/uri/ResourceUri.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.justice.raml.jms.uri;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.apache.commons.lang3.StringUtils.stripStart;
+
+public class ResourceUri {
+    private static final Pattern MESSAGING_RESOURCE_URI_PATTERN =
+            Pattern.compile("^\\/?([a-zA-Z0-9]+)\\.(((api|controller|handler)\\.(commands))|(events))");
```

Comment:
will do it for both regular expressions when your refactoring goes through to the master, e.g. events->event, commands->command etc.
we currently don't support view queries via JMS, so not an issue for now